CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 3.10.0 (August 29, 2025)
+
+BUG FIXES:
+
+* `resource/nsxt_policy_edge_transport_node`: Fix schema set errors in policy edge resource ([#1810](https://github.com/vmware/terraform-provider-nsxt/pull/1810))
+* `resource/nsxt_policy_security_policy`: Handle empty service_entries block case in nsxt_policy_security_policy ([#1798](https://github.com/vmware/terraform-provider-nsxt/pull/1798))
+* `resource/nsxt_policy_tier0_gateway_interface`: Add edge_cluster_member_index to T0 gw interface ([#1737](https://github.com/vmware/terraform-provider-nsxt/pull/1737))
+* `resource/nsxt_policy_host_transport_node_collection`: Replace sub_cluster_id in host_transport_node_collection with sub_cluster_path ([#1733](https://github.com/vmware/terraform-provider-nsxt/pull/1733)) 
+* Allowed usage of default project in various resources and data sources.
+* Various documentation fixes.
+
+IMPROVEMENTS:
+
```

Comment:
Can you please also include https://github.com/vmware/terraform-provider-nsxt/pull/1708?